extensions/amp-social-share/0.1/test/test-amp-social-share.js

```diff
@@ -193,6 +193,19 @@ describes.realWin(
       });
     });
 
+    it('adds a default value for aria-label', () => {
```

Comment:
Are we also validating `aria-label` is overridden correctly... "If a non-empty value is passed in to `aria-label` it will overwrite the default value."

Reply:
Thanks, @kristoferbaxter .   Added a test for the negative case.